modules/nw-ingress-converting-http-header-case.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * ingress/ingress-operator.adoc
+// * upgrade/
+
+[id="nw-ingress-converting-http-header-case_{context}"]
+= Converting HTTP header case
+
+HAProxy 2.2 down-cases HTTP header names by default, for example, changing `Host: xyz.com` to `host: xyz.com`. If legacy applications are sensitive to the capitalization of HTTP header names, use the Ingress Controller `spec.httpHeaders.headerNameCaseAdjustments` API field for a solution to accommodate legacy applications until they can be fixed.
+
+[IMPORTANT]
+====
+Because {product-title} 4.8 includes HAProxy 2.2, make sure to add the necessary configuration by using `spec.httpHeaders.headerNameCaseAdjustments` before upgrading.
+====
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+As a cluster administrator, you can convert the HTTP header case by entering the `oc patch` command or by using the `HeaderNameCaseAdjustments` field by configuring the Ingress Controller YAML file.
+
+Specify a HTTP header to be capitalized by entering the `oc patch` command.
```

Comment:
If you're providing two options (oc patch vs. YAML), these should be unnumbered bullets (not just plain sentences).

Reply:
Also "a HTTP" -> "an HTTP"